app/controllers/RemoveFileController.scala

```diff
@@ -61,7 +61,7 @@ class RemoveFileController @Inject()(upscanService: UpscanService,
               }
             }.recover {
               case _ => {
-                PagerDutyHelper.log("removeFileUpscan", FILE_REMOVAL_FAILURE_UPSCAN)
+                PagerDutyHelper.log("[RemoveFileController][onSubmit]", FILE_REMOVAL_FAILURE_UPSCAN)
```

Comment:
In other files, you've set this as "ClassName Method", in this file you've set it as "[ClassName][Method]", could you do the following so it follows a consistent format: